cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,21 @@
 /// <reference types='cypress' />
+import { LaptopsTab } from '../support/pages/customerUserWay';
 
-describe('', () => {
-  before(() => {
-
+const Laptops = new LaptopsTab();
+describe('Userflow using POM', () => {
+  before('open webpage', () => {
+    Laptops.navigateToHome();
   });
 
-  it('', () => {
-
+  it('full userway', () => {
+    Laptops.clickLaptopTab();
+    Laptops.clickSonyVaio7();
```

Comment:
This is not a good idea to use app content in function name. App content - products, users, etc. - this is content that can be changed today or tomorrow. In this case, better use general naming and pass product name as param